packages/docs-site/src/pages/index.astro

```diff
@@ -0,0 +1,253 @@
+---
+
+import DIcon from '@/components/DIcon.astro';
+import { Aside } from '@astrojs/starlight/components';
+import StarlightPage from '@astrojs/starlight/components/StarlightPage.astro';
+import { DmnoTileLogo } from '@dmno/ui-lib';
+
+import ViteLogoSvg from '~icons/devicon/vitejs?raw';
+import NextLogoSvg from '~icons/logos/nextjs?raw';
+import AstroLogoSvg from '~icons/devicon/astro?raw';
+import NodeLogoSvg from '~icons/logos/nodejs-icon?raw';
+import RollupLogoSvg from '~icons/logos/rollup?raw';
+
+---
+<StarlightPage frontmatter={{ title: '', tableOfContents: false }}>
+  <div class="homepage not-content">
+
+    <div class="hero">
+      <div class="hero__logo">
+          <DmnoTileLogo client:load autoPlay />
+      </div>
+      <div class="hero__text">
+        <h1>DMNO</h1>
+        <h2>The best way to configure, run, and deploy your software</h2>
+        <div class="cta-wrap">
+          <a class="button">Get started</a>
+          <div class="install-command">
+            <DIcon name="console" />
+            npx dmno init
+            <DIcon name="copy" />
+          </div>
+        </div>
+        <!-- <div>
+          <a href={DMNO_PUBLIC_CONFIG.GITHUB_REPO_URL} target="_blank">Star us on GitHub</a>
+        </div> -->
+      </div>
+    </div>
+
+    <!-- <h3>DMNO Config</h3>
+    <p>Helps you define a schema for all of the configuration (environment variables) used throughout your applications.</p> -->
+    <section class="value-prop-section">      
+      <div>
+        <h3>Type-safe env vars</h3>
+        <p>DMNO auto-generates powerful types for your config schema with amazing documentation built right in.</p>
+      </div>
+      <div>
+        <h3>Validate & coerce</h3>
+        <p>Clean and check your config at every step of the way. Our built-in types cover 95% of use cases. Easily add custom validation logic on top.</p>
+      </div>
+      <div>
+        <h3>Monorepo native</h3>
+        <p>DMNO is the only config tool that treats monorepos as a first class concept. Each service can "pick" items from the root or other services.</p>
+      </div>
+      <div>
+        <h3>Live reload</h3>
+        <p>Our `dev` mode reloads your schema and restarts your code whenever anything changes. And it's FAST because it's powered by vite.</p>
```

Comment:
```suggestion
        <p>Our `dev` mode reloads your schema and restarts your code whenever anything changes. And it's FAST because it's powered by Vite.</p>
```